ReactSkia/views/common/RSkTextUtils.cpp

```diff
@@ -77,27 +77,34 @@ void drawText(std::shared_ptr<Paragraph>& paragraph,
             const ParagraphProps& props,
             bool isParent) {
     SkPaint paint;
+    SkPath clipPath;
     SkScalar yOffset = 0;
     Rect frame = layout.frame;
-    auto fontLineHeight = (!std::isnan(props.textAttributes.lineHeight)) && (props.textAttributes.lineHeight >= 0) ?
-                            props.textAttributes.lineHeight :
-                            true;
+    int textHeight=(std::isnan(props.textAttributes.lineHeight)) ? frame.size.height : props.textAttributes.lineHeight;
+    
+    canvas->save();
+    if(textHeight <= 0){
+        return;
```

Comment:
can we add a debug message here to know that we are ignoring text draw because of height

Reply:
I suggest to make this as warning. This case will happen rarely and will happen during the development.